R/g.plot.R

```diff
@@ -176,50 +179,45 @@ g.plot = function(IMP, M, I, durplot) {
     }
     plot(timeline, Acceleration, type = "l", bty = "l",
          lwd = 0.1, axes = FALSE, cex.lab = 0.8,
-         xlab = "", ylab = ylabel,  xlim = c(0, durplot), 
+         xlab = "Recording day", ylab = ylabel,  xlim = c(0, durplot), 
          ylim = YLIM, )
     axis(side = 2, at = YTICKS, las = 1, cex.axis = 0.8)
-    axis(side = 1, at = ticks, labels = tick_labels, las = 3, cex.axis = 0.8)
-    abline(v = noons, lwd = 0.5, col = "grey", lty = 2)
-    abline(v = mnights, lwd = 0.5, lty = 3)
-    lines(timeline, Acceleration, lwd = 1)
     # axis 2 (day counting)
-    if (length(noons) > 1) { # only if more than 1 day
-      axis(side = 1, at = noons, labels = tick2_labels, 
-           cex.axis = 0.8, font = 2, line = -0.4, tick = FALSE)
-      axis(side = 1, at = ticks2, labels = NA, line = 0.5, tck = -0.02)
+    if (length(extranoons) > 1) { # only if more than 1 day
+      axis(side = 1, at = extranoons, labels = x_labels_days, 
+           cex.axis = ifelse(test = length(extranoons) > 10, yes = 0.6, no = 0.8),
+           font = 1, tick = FALSE, lwd = 0.8, mgp = c(3,1,0)) #line = -0.4
+      axis(side = 1, at = ticks_dayborders, labels = NA, tck = -0.05, mgp = c(3,1,0))# line = 0.5, 
     }
   }
-  plot_nonwear = function(timeline, M, durplot, ticks) {
+  plot_nonwear = function(timeline, M, durplot, ticks_12hours) {
     plot(timeline, M$metalong$nonwearscore, type = "s",
          xlab = "", ylab = "Non-wear score", axes = F,
          lwd = 0.1, xlim = c(0,durplot), ylim = c(0, 3), cex.lab = 0.8)
-    axis(side = 2,at = c(0, 1, 2, 3))
-    # axis(side = 1,at = ticks, labels = 1:length(ticks))
+    axis(side = 2,at = c(0, 1, 2, 3), cex.axis = 0.8)
```

Comment:
Maybe add `las = 1` in the axis definition so that y axis is consistent in the lower and upper plots.